homeassistant/components/ovo_energy/sensor.py

```diff
@@ -64,15 +68,32 @@ def __init__(
         key: str,
         name: str,
         icon: str,
-        unit_of_measurement: str = "",
+        device_class: str | None,
+        unit_of_measurement: str | None,
     ) -> None:
         """Initialize OVO Energy sensor."""
+        self._device_class = device_class
         self._unit_of_measurement = unit_of_measurement
 
         super().__init__(coordinator, client, key, name, icon)
 
     @property
-    def unit_of_measurement(self) -> str:
+    def device_class(self) -> str | None:
+        """Return the class of this sensor."""
+        return self._device_class
+
+    @property
+    def last_reset(self) -> datetime | None:
+        """Return the time when the sensor was last reset, if any."""
+        return utc_from_timestamp(0)
+
+    @property
+    def state_class(self) -> str:
+        """Return the state class of this entity."""
+        return "measurement"
```

Comment:
Since both are static, you can also define them in the class definition:

```
_attr_last_reset = utc_from_timestamp(0)
_attr_state_class = "measurement"
```

Reply:
Alternative is using the new SensorEntityDescription